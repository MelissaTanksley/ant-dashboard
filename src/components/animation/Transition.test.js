import React from 'react';
import { shallow } from 'enzyme';
import Transition from './transition';
import toJson from "enzyme-to-json";

describe('<Transition />', () => {
    const wrapper = shallow(<Transition />)
    const div = <div>this is TransitionComponent</div>
    it('Transition matches snapshot', () => {
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})