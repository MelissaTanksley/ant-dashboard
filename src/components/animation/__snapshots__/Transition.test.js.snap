// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Transition /> Transition matches snapshot 1`] = `
<div>
  this is TransitionComponent
</div>
`;
